packages/contentful--react-apps-toolkit/src/useCMA.ts

```diff
@@ -0,0 +1,32 @@
+import { createClient, PlainClientAPI } from "contentful-management";
+import { useEffect, useState } from "react";
+import { useSDK } from "./useSDK";
+
+/**
+ * A react hook returning the plain client CMA.
+ *  - Depends on Apps SDK.
+ *  - Must be called in the SDKProvider component (./SDKProvider). If called outside the provider, will throw error. useSDK hook called on line 12 handles error.
+ */
+
+export function useCMA(): PlainClientAPI | undefined {
+  const sdk = useSDK();
+  const [cma, setCMA] = useState<PlainClientAPI | undefined>(undefined);
+
+  useEffect(() => {
+
+    const client = createClient(
+      { apiAdapter: sdk.cmaAdapter },
+      {
+        type: "plain",
+        defaults: {
+          environmentId: sdk.ids.environment,
+          spaceId: sdk.ids.space,
+        },
+      }
+    );
+
+    setCMA(client);
+  }, [sdk]);
```

Comment:
Can we use `useMemo` here? No need for `useEffect`

Reply:
Implemented with use memo, but maybe have a quick look to make sure I've done it right :)